plugins/hue/fetch_url_v2.py

```diff
@@ -0,0 +1,59 @@
+    # hier die abgefangenen fehlermeldungen in den connections, die auf das fehleritem gemapped werden
+    self._connErrors = ['Host is down', 'timed out', '[Errno 113] No route to host']
+
+
+    def _fetch_url_v2(self, url, auth=None, username=None, password=None, timeout=2, method='GET', headers={}, body=None, errorItem=None):
```

Comment:
Was my first idea to change it in tools. I'm going to prepare this change to tools.py nod remove from this plugin in a next pull request. 
Con: I have currently no clue who isn't using fetch_url and I have only limited testing for it. The hue case works fine.
I give it a try, otherwise we have to fix the bugs.